src/main/java/org/alliancegenome/curation_api/services/ConditionRelationService.java

```diff
@@ -71,4 +78,35 @@ public ConditionRelation getStandardExperiment(String experimentName, Reference
 		conditionRelation.setConditions(List.of(condition));
 		return conditionRelation;
 	}
+
+	public SearchResponse<ConditionRelation> getConditionRelationSearchResponse(HashMap<String, Object> params) {
+		String key = "singleReference.curie";
+		String referenceID = (String) params.get(key);
+		if (StringUtils.isEmpty(referenceID)) {
+			ObjectResponse<ConditionRelation> response = new ObjectResponse<>();
+			response.addErrorMessage(key, "Cannot find any reference ID under map key: " + key);
+			throw new ApiErrorException(response);
+		}
+		Reference reference = referenceService.get(referenceID).getEntity();
```

Comment:
You can probally just use the DAO here instead of the service but whatever.

Reply:
Maybe. I'll leave it for now.